pyproject.toml

```diff
@@ -135,7 +135,7 @@ tests/codegen/snapshots/
 '''
 
 [tool.pytest.ini_options]
-addopts = "-s --emoji --mypy-ini-file=mypy.ini --benchmark-disable"
+addopts = "-s --emoji --mypy-ini-file=mypy.ini --benchmark-disable -n auto"
```

Comment:
this would always run them in parallel, not sure if we want that by default? 

I don't have a strong opinion, but when developing I usually run a few test at once so parallel might be an overkill

Reply:
hmm what ever you want I don't really care, it is just anoying to add -n auto in pycharm configuations..